src/containers/projects/new/step-1.svelte

```diff
@@ -0,0 +1,86 @@
+<script>
+  import { fade } from 'svelte/transition';
+  import BottomNavigation from '@/components/projects/new/bottom-navigation.svelte';
+  import StepHeader from '@/components/projects/new/step-header.svelte';
+  import FormField from 'ui/form-field.svelte';
+  import TextField from 'ui/text-field.svelte';
+  import Button from 'ui/button.svelte';
+  import Dropzone from 'ui/dropzone.svelte';
+  import {project, changeField, save} from '@/store/new-project';
+  import {readFileAsync} from '@/utils/read-files';
+
+  $: file = $project.image;
+  let image = null;
+  $: (async() => image = file ? await readFileAsync(file) : null)();
+
+  const addImage = async e => changeImageAndSave(e.detail[0]);
```

Comment:
why `async`?

Reply:
Because we have to resolve the promise

Reply:
Which promise? `changeImageAndSave` is a synchronous function, and you're not using any `await`s

Reply:
Oh, sorry, I was looking at the readFileAsync function above. Right